src/models/data_models/image.ts

```diff
@@ -1,8 +1,12 @@
 import { Data, Extracted, Extraction } from '../data';
 
+export const enum ImageProviderName {
+	yelp = 'YELP',
+	google_places = 'GOOGLE PLACES',
+}
 // A picture of something to do with a business
 export interface Image extends Data {
-	// TODO (Benny): probably a bunch of other fields belong here
+	provider: ImageProviderName;
```

Comment:
Reminder for myself to update the mock extraction provider to include some hard coded provider after this is merged